nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -62,10 +63,10 @@ class BasePruningAlgoBuilder(TFCompressionAlgorithmBuilder):
 
     def __init__(self, config: NNCFConfig, should_init: bool = True):
         super().__init__(config, should_init)
-        params = config.get('params', {})
+        params = self._algo_config.get('params', {})
         self._params = params
-        self.ignored_scopes = self.config.get('ignored_scopes', [])
-        self.target_scopes = self.config.get('target_scopes')
+        self.ignored_scopes = self._algo_config.get('ignored_scopes', [])
+        self.target_scopes = self._algo_config.get('target_scopes')
```

Comment:
`self.ignored_scopes` and `self.target_scopes` is initialized in the `BaseCompressionAlgorithmBuilder` and you should not reimplement it here.

Reply:
Thanks for spotting this, fixed.